batch/batch/front_end/front_end.py

```diff
@@ -2207,9 +2198,11 @@ async def close_project(tx):
 LEFT JOIN batches
 ON billing_projects.name = batches.billing_project
 AND billing_projects.`status` != 'deleted'
-AND batches.time_completed IS NULL
 AND NOT batches.deleted
+LEFT JOIN batch_updates
+ON batches.id = batch_updates.batch_id
 WHERE name = %s
+AND (batches.time_completed IS NULL OR NOT batch_updates.committed)
```

Comment:
I think we can just do an aggregation here to count number of rows where committed is false. All batches should have updates at this point. This will future proof it for when we take out `time_completed` from the batches table. I thought I did that in this set of PRs to not use `time_completed` from the batches table but instead compute it from batch_updates but maybe not.

Reply:
Don't we also want to disallow closure of a billing project while a batch is running? So we would either check the state of the batch or the time_completed